tools/single_use/Normalize_Consent_Data.php

```diff
@@ -99,8 +99,8 @@
         $consentID  = $entry['ID'];
         $candID     = $entry['CandID'];
         // Check if consent status is not_answered
-        if($status === "not_answered") {
-            array_push($errors, "Deprecated consent status 'not_answered' found for " . $consentName . ":
+        if($status === "not_answered" || $status === "") {
```

Comment:
This doesn't look like it's in the right PR?

Reply:
This still needs to be removed!